packages/shared/friends/sagas.ts

```diff
@@ -796,9 +799,17 @@ function sendUpdateUserStatus(id: string, status: CurrentUserStatus) {
 
 function updateUserStatus(client: SocialAPI, ...socialIds: string[]) {
   const statuses = client.getUserStatuses(...socialIds)
+  const lastStatuses = getLastStatusOfFriends(store.getState())
 
   statuses.forEach((value: CurrentUserStatus, key: string) => {
-    sendUpdateUserStatus(key, value)
+    const lastStatus = lastStatuses.get(key)
+    if (!lastStatus || !deepEqual(lastStatus, value)) {
+      sendUpdateUserStatus(key, value)
+      lastStatuses.set(key, value)
+    } else {
+      // avoid sending already sent states
+      logger.log(`friend ${key} status was ignored - duplicated state`)
```

Comment:
Let's set this to `trace` instead:
```suggestion
      logger.trace(`friend ${key} status was ignored - duplicated state`)
```

Reply:
I'd remove the trace/log from the PR before merging, to reduce the noise on the console and to prevent memory errors of these logs which in normal sessions are never garbage collected

Reply:
Didn't see this comment. Okay, understood! Just to know: In which sessions are the logs garbage collected?

Reply:
That only happens if you clear the logs. Or depends entirely on the browser if the console is disabled.

For that reason we add DEBUG_COMMS to enable the comms logger as an example